src/main/java/hudson/plugins/sloccount/SloccountBuildAction.java

```diff
@@ -101,11 +115,33 @@ SloccountBuildAction getPreviousAction(){
         return null;
     }
 
-    public AbstractBuild<?,?> getBuild(){
+    public Run<?,?> getBuild(){
```

Comment:
I followed one of blog post (I think it was for Gatling plugin) - there was info that AbstractBuild and AbstractProject are converted to Run and Job.